fmriprep/interfaces/confounds.py

```diff
@@ -487,7 +487,7 @@ def _run_interface(self, runtime):
             remap_rois=False,
             labels=(
                 ("WM+CSF", "Edge") if has_cifti else
-                ("Ctx GM", "dGM", "WM+CSF", "Cb", "Edge")
+                ("Ctx GM", "dGM", "shallow WM+CSF", "deep WM+CSF", "Cb", "Edge")
```

Comment:
What is `dGM`? Deep gray matter? Should we be referencing this anywhere like in the figure caption?

And if so, maybe this could become

```suggestion
                ("Ctx GM", "dGM", "sWM+sCSF", "dWM+dCSF", "Cb", "Edge")
```

Or something.

cc @oesteban

Reply:
Seems pretty effective to me :)